pkg/api/api.go

```diff
@@ -343,8 +343,9 @@ func (hs *HTTPServer) registerRoutes() {
 			alertsRoute.Get("/states-for-dashboard", Wrap(GetAlertStatesForDashboard))
 		})
 
-		apiRoute.Get("/alert-notifications", Wrap(GetAlertNotifications))
-		apiRoute.Get("/alert-notifiers", Wrap(GetAlertNotifiers))
+		apiRoute.Get("/alert-notification-channels", Wrap(GetAlertNotificationChannels))
```

Comment:
The http api for alert notifications/channels is totally messed up so didn't now what to call this new endpoint. Any better suggestions/ideas?

Reply:
feels a bit messy to have it as a new top level URI resource name. 
* /alert-notifications/read-only
* /alert-notifications/basic-list

Reply:
Yeah, I totally agree. For the /org/users I'm thinking of adding a `/org/users/lookup` so maybe `/alert-notifications/lookup` can work?

Reply:
yes!